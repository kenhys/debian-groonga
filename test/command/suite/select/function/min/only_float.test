table_create Values TABLE_NO_KEY
column_create Values output COLUMN_SCALAR Float

load --table Values
[
{}
]

select Values \
  --filter true \
  --output_columns '_id,output' \
  --scorer 'output = min(-29.29, -2929.29, 29.29)'
