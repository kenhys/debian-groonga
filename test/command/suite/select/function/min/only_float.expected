table_create Values TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Values output COLUMN_SCALAR Float
[[0,0.0,0.0],true]
load --table Values
[
{}
]
[[0,0.0,0.0],1]
select Values   --filter true   --output_columns '_id,output'   --scorer 'output = min(-29.29, -2929.29, 29.29)'
[[0,0.0,0.0],[[[1],[["_id","UInt32"],["output","Float"]],[1,-2929.29]]]]
